tools/packaging/packaging.psm1

```diff
@@ -379,6 +379,7 @@ function Start-PSPackage {
                     Version = $Version
```

Comment:
@TravisEz13, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v7.0.0-rc.2/PowerShell-7.0.0-rc.2-win-x64.msi should work
```powershell
Unable to read data from the transport connection: Connection reset by peer.
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 114
114:                                 $null = Invoke-WebRequest -uri $url -RetryIntervalSec 10 -MaximumRetryCount 6

```